src/load_stats.py

```diff
@@ -16,36 +63,25 @@ def load_stats(output_file):
     #
 
     stats = {site: dict() for site in config.STATS_SITES}
+    pool = multiprocessing.Pool()
 
-    for i_file, dump_file in enumerate(config.STATS_DUMP_FILES):
-        path = os.path.join(config.STATS_DUMP_DIR, dump_file)
-        print('({}/{})'.format(i_file + 1, len(config.STATS_DUMP_FILES)), path)
-
-        try:
-            f = gzip.open(path, 'r')
-        except FileNotFoundError as e:
-            print('Skipped:', e, file=sys.stderr)
-            continue
-
-        for i_line, line in enumerate(gzip.open(path)):
-            try:
-                #  `responses` is ignored as it appears to always be 0
-                site, page, views, _responses = line.decode().strip().split()
-            except Exception as e:
-                print('Error at line', i_line, ':', e, file=sys.stderr)
-                continue
-
-            page = page.replace('_', ' ')
+    for i_file, file_stats in enumerate(pool.imap(load_file, files)):
+        print(
+            "({}/{}) {}: updating {} items".format(
+                i_file + 1,
+                len(files),
+                files[i_file],
+                sum(len(lang) for lang in file_stats.values()),
+            )
+        )
 
-            if site not in config.STATS_SITES:
-                continue
-
-            if page not in stats[site]:
-                stats[site][page] = int(views)
-            else:
-                stats[site][page] += int(views)
+        for lang, pages in file_stats.items():
+            for page, views in pages.items():
+                stats[lang].setdefault(page, 0)
```

Comment:
Initializing `stats` with [`defaultdict`](https://docs.python.org/3/library/collections.html#collections.defaultdict) seems to be a bit more efficient